app/script/storage/StorageService.coffee

```diff
@@ -174,20 +182,29 @@ class z.storage.StorageService
           $.extend event, mapped_event
       @db.version(8).stores version_5
       .upgrade (transaction) =>
-        @logger.warn "Database upgrade to version #{@db.verno}", transaction
+        @logger.warn 'Database upgrade to version 8', transaction
         transaction[@OBJECT_STORE_CONVERSATION_EVENTS].toCollection().modify (event) ->
           if event.type is z.event.Client.CONVERSATION.DELETE_EVERYWHERE
             event.time = new Date(event.time).toISOString()
       @db.version(9).stores version_9
       @db.version(10).stores version_10
       .upgrade (transaction) =>
-        @logger.warn "Database upgrade to version #{@db.verno}", transaction
+        @logger.warn 'Database upgrade to version 10', transaction
         transaction[@OBJECT_STORE_CONVERSATION_EVENTS].toCollection().modify (event) ->
           event.category = z.message.MessageCategorization.category_from_event event
+      @db.version(11).stores version_11
```

Comment:
Client update to make sure that database very old clients look the same as newer ones moving forward.